### ISSUE TITLE
[INTAKE] Confusing heading order on all hub pages, reported by a Veteran who uses a screen reader

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#132452]
# Content and IA Product Support Request

The VA 508 Support email account received feedback from a Veteran who uses a screen reader that the hub pages like [this one](https://www.va.gov/health-care/about-va-health-benefits/) are confusing since the heading order is unexpected and confusing. 

The Veteran was confused by the "Health care" H4 being the first thing on the page that they heard with their screen reader. 

<img width="1060" height="948" alt="Image" src="https://github.com/user-attachments/assets/6cfcd7ef-af6f-4873-94dc-ec6d8a15780c" />

The text description of the headings map image above:

            4 Health care
1 About VA health benefits
    2 On this page
    2 What care and services does VA health care cover?
    2 What if I’m just separating from active duty and don’t know where to start?
    2 How can I get my IRS 1095-B tax form as a Veteran?
    2 More about your medical benefits package
        3 What basic health care services does VA cover?
        3 What other medical services will VA cover to support my treatment?
...

## Team info

- **Product name:** Hub pages (is this public websites team?)
- **Team name:** Accessibility (ADE)
- **Slack channel:** #accessibility-help
- **OCTO product owner:** @artsymartha68 @DanielleThierryUSDSVA @RLHecht 

### What audiences is your work for
- [x] Veterans and/or their family members or caregivers
- [ ] Other VA.gov audience
  - Please describe the audience:
- [ ] Internal designers and product team members (i.e. pattern or standards work)

### Which of these descriptions best fits the work we’ll partner on?

- [ ] Digitizing a new form
- [ ] Creating a new digital tool for Veterans or their beneficiaries
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [x] Components, patterns, or general standards
- [x] Something else: All hub page templates are affected by this change

### Describe the work effort

The accessibility folks had a long chat about it recently, and, in short, we think that the [standard side nav component with categorized links](https://design.va.gov/components/sidenav) should be used instead. Per the sample page, the Profile that appears to be a header is coded as a `<span>`. That will prevent it from appearing in the heading map. 

<img width="2266" height="978" alt="Image" src="https://github.com/user-attachments/assets/0735b4f6-d3a5-4f66-8d25-a551e19af808" />

  
### Timeline

- **Estimated launch timeline:** As soon as possible, would prefer a higher priority since this was reported by a Veteran

## Acceptance criteria
- [ ]  All Health care benefit hub detail pages with a left nav: 'Health care' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`

  - Example [page](https://www.va.gov/health-care/about-va-health-benefits/)
- [ ]  All Family and caregiver benefits benefit hub detail pages with a left nav:  'Family and caregiver benefits' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](hhttps://www.va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/)
- [ ]  All Disability benefits benefit hub detail pages with a left nav:  'Disability benefits' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/disability/upload-supporting-evidence/)
- [ ]  All Education and training benefit hub detail pages with a left nav:  'Education and training' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/education/verify-school-enrollment/)
- [ ]  All Careers and employment benefit hub detail pages with a left nav:  'Careers and employment' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/careers-employment/education-and-career-counseling/)
- [ ]  All Pension benefit benefit hub detail pages with a left nav:  'Pension benefit' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/pension/eligibility/)
- [ ]  All Housing assistance benefit hub detail pages with a left nav:  'Housing assistance' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/housing-assistance/home-loans/check-coe-status/)
- [ ]  All Life insurance benefit hub detail pages with a left nav:  'Life insurance' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/life-insurance/totally-disabled-or-terminally-ill/)
- [ ]  All Burials and memorials benefit hub detail pages with a left nav:  'Burials and memorials' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/burials-memorials/pre-need-eligibility/)
- [ ]  All Records benefit hub detail pages with a left nav:  'Records' displayed above the left nav is now a span with utility classes to match the visual look of the H4, and is no longer a semantic heading `<h4>`
  - [ ] Example [page](https://www.va.gov/records/download-va-letters/)
- [ ] screen reader users are still able to access it, but it won't be considered a heading
- [ ] A11y review by @qjac 
- [ ] A11ly review by @coforma-jamie

**Comment by qjac** [comment]
@FranECross should those pages be checked in prod or is there a review instance?
